arangod/Indexes/IndexFactory.cpp

```diff
@@ -76,6 +76,30 @@ struct InvalidIndexFactory : public IndexTypeFactory {
 }  // namespace
 
 namespace arangodb {
+namespace helpers {
+
+IndexId extractId(velocypack::Slice slice) noexcept {
+  auto id = IndexId::none();
+  auto value = slice.get(StaticStrings::IndexId);
+  if (value.isNumber()) {
+    id = IndexId{value.getNumber<IndexId::BaseType>()};
+  } else if (value.isString()) {
+    IndexId::BaseType base;
+    if (absl::SimpleAtoi(value.stringView(), &base)) {
+      id = IndexId{base};
+    }
+  }
+  return id;
+}
+
+std::string_view extractName(velocypack::Slice slice) noexcept {
+  if (auto value = slice.get(StaticStrings::IndexName); value.isString()) {
```

Comment:
Same here: if `slice` is not of type `Object`, the `get` will throw.

Reply:
slice always object